docs/conf.py

```diff
@@ -0,0 +1,61 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Schematic'
+copyright = '2021, Sujay Patil'
+author = 'Sujay Patil'
+
+# The full version, including alpha/beta/rc tags
+release = '1.0.0'
```

Comment:
Can you look into whether you can update this section using the contents of `pyproject.toml`? It shouldn't be too hard since this is a Python script. There might also be precedent for this online.

Reply:
This is an optional request.